Detectors/util/pcb-layers.nbt

```diff
@@ -4,16 +4,21 @@ fn _hexagon_area(side: Length) -> Area = 3*sqrt(3)/2*side*side
 fn _copper_layer_weight(n: Scalar) = ((n-2)*(0.5 oz / ft^2) + 2*(1.0 oz / ft^2))
 
 let hexagon_side : Length = 85 mm
-let hexagon_thick : Length = 1.66 mm
+let pcb_thick : Length = 1.2 mm
```

Comment:
While I like numbat, I think that it is more transparent to future developers to have this calculation inside of `ecal_layer_stack.py` as well. So I'm going to remove `pcb-layers.nbt` and do the estimate of the copper thickness in a PCB layer "live" in the Python script.